locales/pl/ways-to-give.properties

```diff
@@ -0,0 +1,16 @@
+# Ways to give page
+ways_to_give=Sposoby wsparcia
+credit_debit_paypal=Karta kredytowa/debetowa i PayPal
+make_donation=<a href="https://donate.mozilla.org/&ref=EOYFR2015&utm_campaign=EOYFR2015&utm_source=wiki.mozilla.org&utm_medium=referral&utm_content=Ways_to_Give">Pod tym adresem</a> można bezpiecznie wpłacić datek przez Internet.
+mail_check_heading=Czek (pocztą)
+mail_check_body=Można wystawić czek na „Mozilla Foundation, Inc.”. Prosimy nie wysyłać pocztą gotówki. Adres do wysłania czeku:
+canadian_check_cost_info=Wszystkie czeki wystawione w dolarach kanadyjskich podlegają opłacie manipulacyjnej banku w wysokości 5 dolarów, nałożonej Mozilli za każdy czek przechodzący przez nasz bank.
+foreign_check_cost_info=Jeśli mieszkasz poza Stanami Zjednoczonymi lub Kanadą i chcesz wysłać nam czek w walucie innej niż dolar amerykański lub kanadyjski, to prosimy pamiętać, że nasz bank pobiera od nas opłatę w wysokości czterdziestu pięciu dolarów amerykańskich za każdy czek wystawiony w walucie innej niż dolar amerykański lub kanadyjski.
+currencies=Waluty
+select_donation_currency=Wybierz walutę, aby wykonać bezpieczny przelew przez Internet:
```

Comment:
"Wybierz walutę, aby w bezpieczny sposób przekazać datek przez Internet:" would be closer to en-US ("Select your currency to make a secure online donation:")